javascript-data-model/order-history.js

```diff
@@ -0,0 +1,88 @@
+var books = [
+  {
+    title: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    returnWindow: 'Sep 7, 2020',
+    subtotal: '$31.55'
+  },
+
+  {
+    title: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    returnWindow: 'Aug 19, 2020',
+    subtotal: '$41.33'
+  },
+
+  {
+    title: 'The Art of Sql',
+    author: 'Faroult Stephane',
+    returnWindow: 'Aug 4, 2020',
+    subtotal: '$33.99'
+  }
+];
+
+var electronics = [
+  {
+    product: 'Gamecube Controller Adapter',
+    returnWindow: 'Aug 5, 2020',
+    subtotal: '$15.98'
+  },
+
+  {
+    product: 'Gamecube Controller SSB edition',
+    returnWindow: 'Aug 4, 2020',
+    subtotal: '$94.95'
+  }
+];
+
+var products = {
+  category: [books, electronics]
+};
```

Comment:
Rather than storing this data in separate variables, please write it directly into the `orderHistory` variable's content so I can easily read your data as a whole.